presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveCreateExternalTable.java

```diff
@@ -54,22 +54,22 @@ public void testCreateExternalTableWithData()
         File tableLocation = new File(tempDir, "data");
 
         @Language("SQL") String createTableSql = format("" +
-                        "CREATE TABLE test_create_external " +
+                        "CREATE TABLE test_create_external_with_data " +
```

Comment:
> Rename test databases to prevent name collisions

-> `Rename test tables ...`

Reply:
Doh, good point - - It was the tables that were renamed.

I just rebased this PR on tip master, since there were conflicts and removed the commit that renamed the tables.  If the Hive test failures recur, I'll submit a separate PR with the renames.